en/sql/dblink.rst

```diff
@@ -642,23 +642,199 @@ The Query statement below is a Query statement that inquires the remote_t table
+    there’s no result
+
+*   TRUNCATE statement is not supported.
+*   CREATE TABLE … LIKE table@server syntax is not supported (For reference, CREATE TABLE … AS SELECT FROM table@server syntax is supported)
+*   dblink() and remote table (@server) are not supported in the TRIGGER statement.
+*   predicate push: The SELECT statement written in table extension format (@server) syntax is internally rewritten as DBLINK() syntax. To improve performance, it is rewritten by pushing conditional clauses that can be processed into the query to be performed in the remote DB. However, if a built-in function or user-defined function in a conditional clause is used, it is excluded from push.
+*   Performance notes
+
+    .. note::
+        When the connect by clause, group by clause, having clause, and limit clause are used in the SELECT statement in table extension format (@server), the where condition, group by clause, having clause, and limit clause are not executed in the remote DB. After tranferring to the local DB, performance may be slowed by performing tasks that meet the conditions.
+
+
+    Below is an example of processing the group by clause after transferring all data from the remote DB's tree table to the local DB in order to process count().
+
+    .. code-block:: sql
+
+        -- original query
+        SELECT A.parentid, count()
+        FROM tree@srv1 A
+        GROUP BY A.parentid ;
+        -- rewritten query
+        SELECT A.parentid, count()
+        FROM ( SELECT [_dbl].parentid
+               FROM DBLINK( srv1 /* '192.168.1.125:33000:remotedb1:dba::' */,
+                            'SELECT parentid FROM tree A'
+                          ) AS [_dbl](parentid integer)
+             ) A (parentid)
+        GROUP BY A.parentid
+
+*   The SYSDATE function used in the table extension format (@server) syntax is performed in the local DB, so caution is required if the time between servers is different.
+
+    .. code-block:: sql
+
+        -- original query
+        SELECT * FROM tbl@srv1 WHERE col1 >= sysdate;
+        -- rewritten query
+        SELECT *
+        FROM ( SELECT col1, col2
+               FROM DBLINK( srv1 /* '192.168.1.125:33000:remotedb1:dba::' */,
+                            'SELECT col1, col2 FROM tbl'
+                          ) AS [_dbl](col1 date, col2 varchar)
+             ) tbl (col1, col2)
+        WHERE col1>= SYS_DATE
+
+*   When using a scalar subquery with a co-related condition in the table extension format (@server) syntax, a subquery, or a SELECT query with an EXIST statement, the remote query brings the entire data to the local DB every time and performs the task of finding data corresponding to the join column. As a result, a rapid decrease in performance occurs. The example below uses T1.a as a condition for a scalar subquery, and is a query that finds suitable data by bringing all the data from svr1's tree table as much as T1.a < 4 to the local DB.

```

Comment:
```suggestion
*   When using a scalar subquery, subquery and EXIST clause  with a co-related condition in the table extension format (@server),  the remote query brings the whole data to the local DB every time and performs to find data corresponding to the join column.  As a result, a rapid decrease in performance occurs. The example below uses T1.a as a condition for a scalar subquery, and all data as T1.a < 4 of svr1's tree table is sent to the local DB to find suitable data per executing scalar subquey, so it may slow performance.

```